python/mxnet/rnn/rnn_cell.py

```diff
@@ -284,10 +304,14 @@ def unroll(self, length, inputs, begin_state=None, layout='NTC', merge_outputs=N
 
         Returns
         -------
-        outputs : list of Symbol
-            output symbols.
+        outputs : list of Symbol or Symbol
+            Symbol (if `merge_outputs` is True) or list of Symbols
```

Comment:
Good idea - Do we want to do this now for this case or go over and fix all of these cases in a separate iteration?